src/pal/src/exception/seh.cpp

```diff
@@ -30,18 +28,19 @@ Module Name:
 #include "pal/init.h"
 #include "pal/process.h"
 #include "pal/malloc.hpp"
+#include "signal.hpp"
 
 #if HAVE_ALLOCA_H
-#include "alloca.h"
+#  include "alloca.h"
```

Comment:
Please don't indent these. We don't use this convention anywhere in PAL / CoreCLR